### PR TITLE
Add MHO-C401 BLE smart hygrometer support

### PR DESCRIPTION
**Pull request by vevsvevs**
## Description:

**Related issue (if applicable):** esphome/feature-requests#808

**Pull request in [esphome-docs](https://github.com/esphome/esphome-docs) with documentation (if applicable):** esphome/esphome-docs#806

## Checklist:
  - [X] The code change is tested and works locally.
  - [ ] Tests have been added to verify that the new code works (under `tests/` folder).

If user exposed functionality or configuration variables are added/changed:
  - [ ] Documentation added/updated in [esphome-docs](https://github.com/esphome/esphome-docs).

### CODE REVIEW

**Review by nikito7 — APPROVED** [review]

**Comment by nikito7** [comment]
When merged?

File: esphome/components/xiaomi_mhoc401/sensor.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import sensor, esp32_ble_tracker
+from esphome.const import CONF_BATTERY_LEVEL, CONF_HUMIDITY, CONF_MAC_ADDRESS, CONF_TEMPERATURE, \
+    UNIT_CELSIUS, ICON_THERMOMETER, UNIT_PERCENT, ICON_WATER_PERCENT, ICON_BATTERY, CONF_ID, \
+    CONF_BINDKEY
+
+DEPENDENCIES = ['esp32_ble_tracker']
+AUTO_LOAD = ['xiaomi_ble']
```
Comment (jesserockz) [inline esphome/components/xiaomi_mhoc401/sensor.py:9]: Please add yourself as a codeowner and run `script/build_codeowners.py`
```suggestion
CODEOWNERS = ['@vevsvevs']
DEPENDENCIES = ['esp32_ble_tracker']
AUTO_LOAD = ['xiaomi_ble']
```

**Review by jesserockz — CHANGES_REQUESTED** [review]

**Comment by vevsvevs** [comment]
@jesserockz I've applied suggested changes, but now check fails with `CODEOWNERS file is not up to date`. Running locally `script/build_codeowners.py` also fails for me, so now I'm stuck :(

**Comment by nikito7** [comment]
New PR here: https://github.com/esphome/esphome/pull/1486

**Pull request closed without merging**